README.md

```diff
@@ -75,6 +73,7 @@ projects:
 - `refresh` is the time to wait for refresh the slot page in seconds (default: 30)
 - `range` is the number of days during which you are looking for slots (default: 7)
 - `disponibility` is the range of the hours you want to be alerted (default: 00:00-23:59)
+- `avoid_spam: if true, only one message will be sent for a given slot
```

Comment:
the markdown is broken on avoid_spam, need to close the backticks

Reply:
done!